include/swift/SILOptimizer/Utils/OwnershipOptUtils.h

```diff
@@ -27,6 +27,12 @@
 
 namespace swift {
 
+/// Returns true if this value requires OSSA cleanups.
+inline bool requiresOSSACleanup(SILValue v) {
+  return v->getFunction()->hasOwnership()
+    && v.getOwnershipKind() != OwnershipKind::None;
```

Comment:
What about OwnershipKind::Unowned?

Reply:
I agree. At the time I didn't want to risk having optimizations kick in for Unowned stuff.